odoo/service/db.py

```diff
@@ -211,9 +211,13 @@ def dump_db(db_name, stream, backup_format='zip'):
             return stdout
 
 def exp_restore(db_name, data, copy=False):
+    def chunks(d, n=8192):
```

Comment:
Since we don't use `exp_restore` anymore in the scope of this PR, this modification could be dropped but IIRC `exp_restore` is still used in the restful api and this chunking system addresses a critical point of memory usage where the whole database was loaded into RAM by `.decode('base64')`